### PR TITLE
Compression optimization to reduce storage size.

### LINKED ISSUE #39: Too much storage required
Hello everyone, the video2tfrecord library is working very well, great work!  

I've done just some small modifications to run the code on TF 2, and this is one of my suggestions, we could detect the TensorFlow version and change the code dynamically. 

 Another interesting point is the compression of the video frames. Using the default code, each TFRecord with 5 videos and 25 frames got about 340MB. With this proportion, just my test set would get about 21GB...

To solve this, I changed the code to generate the byte image from this: ```image.tostring()``` to this: ```tf.image.encode_jpeg(image).numpy()```. With this change, the same TFRecord just got about 18MB which is much more efficient to store video datasets. This could be a second modification...

What do you think about this?

### PR DESCRIPTION
**Pull request by AlvaroCavalcante**
This PR closes #39 where the main idea is to try to reduce TFRecord size. 

I added a hyperparameter called "jpeg_encode" that is true by default, once I believe that the user may want to run this script with this configuration. Based on this, we set the encoding type to jpeg instead of transforming the image into a simple string, which uses a lot of storage space. 

Besides that, I also added verification of the TensorFlow version, to avoid errors when using TF > 2.

The jpeg decoder was also added into the test file to allow the user to decode the image correctly, in the case of jpeg encode.

Finally, there's an important point that I would like to talk about. I realized that you use the Numpy arrays of type "uint32". I would like to know more about this choice because the "uint8" type is enough to store images (where the value is between 0 and 255) and this kind of array is smaller than the other one, which occupies less memory and storage. 

So, do you think that we could change the type of the array? Or the "uint32" is important for some reason? Thank you.

### CODE REVIEW

**Comment by ferreirafabio** [comment]
Thank you! :-)
